src/System.Net.Http/tests/FunctionalTests/HttpClientHandlerTest.Authentication.cs

```diff
@@ -522,6 +522,41 @@ public static IEnumerable<object[]> ServerUsesWindowsAuthentication_MemberData()
 
         private static bool IsNtlmInstalled => Capability.IsNtlmInstalled();
         private static bool IsWindowsServerAvailable => !string.IsNullOrEmpty(Configuration.Http.WindowsServerHttpHost);
+        private static bool IsDomainJoinedServerAvailable => !string.IsNullOrEmpty(Configuration.Http.DomainJoinedHttpHost);
+
+        [ConditionalFact(nameof(IsDomainJoinedServerAvailable))]
+        public async Task Credentials_DomainJoinedServerUsesKerberos_Success()
+        {
+            if (IsCurlHandler)
+            {
+                throw new SkipTestException("Skipping test on CurlHandler (libCurl)");
```

Comment:
I haven't seen this before. Is there a new pattern for skipping tests?

Reply:
Yes.  It requires ConditionalFact or ConditionalTheory even if you don't specify a condition.  And it requires another using statement as well.

```c#
using Microsoft.DotNet.XUnitExtensions;
```

We've started to use this pattern in our updated tests.

Reply:
And using this new pattern shows up better in the test logs because it will that it skipped the test and print out the reason why.